docs/howto/static-files/index.txt

```diff
@@ -19,7 +19,8 @@ Configuring static files
       STATIC_URL = 'static/'
 
 #. In your templates, use the :ttag:`static` template tag to build the URL for
-   the given relative path using the configured :setting:`STATICFILES_STORAGE`.
+   the given relative path using the configured ``staticfiles`` storage from
+   :setting:`STORAGES`.
```

Comment:
Here, and in the other locations where "``staticfiles`` storage from :setting:`STORAGES`" appears, I wonder if we can't say more naturally, "``staticfiles`` :setting:`STORAGES` alias"?


```suggestion
   the given relative path using the configured ``staticfiles``
   :setting:`STORAGES` alias.
```